plutus-pab/src/Plutus/PAB/Webserver/API.hs

```diff
@@ -19,37 +17,31 @@ import           Plutus.PAB.Webserver.Types (ContractActivationArgs, ContractIns
                                              ContractSignatureResponse, FullReport)
 import           Servant.API                (Capture, Get, JSON, Post, Put, ReqBody, (:<|>), (:>))
 import           Servant.API.WebSocket      (WebSocketPending)
-import           Wallet.Types               (ContractInstanceId, NotificationError)
+import           Wallet.Types               (ContractInstanceId)
 
 type WalletProxy walletId = "wallet" :> (Wallet.API walletId)
 
-type API t
-     = "api" :> ("healthcheck" :> Get '[ JSON] ()
-                 :<|> "full-report" :> Get '[ JSON] (FullReport t)
-                 :<|> "contract" :> ("activate" :> ReqBody '[ JSON] t :> Post '[ JSON] ContractInstanceId
-                                     :<|> Capture "contract-instance-id" Text :> ("schema" :> Get '[ JSON] (ContractSignatureResponse t)
-                                                                                  :<|> "endpoint" :> Capture "endpoint-name" String :> ReqBody '[ JSON] JSON.Value :> Post '[JSON] (Maybe NotificationError))))
-
 type WSAPI =
     "ws" :>
         (Capture "contract-instance-id" ContractInstanceId :> WebSocketPending -- Websocket for a specific contract instance
         :<|> WebSocketPending -- Combined websocket (subscription protocol)
         )
 
--- | PAB client API for contracts of type @t@. Examples of @t@ are
---   * Contract executables that reside in the user's file system
+-- | PAB client API for contracts of type @t@. An example of @t@ are
 --   * "Builtin" contracts that run in the same process as the PAB (ie. the PAB is compiled & distributed with these contracts)
-type NewAPI t walletId -- see note [WalletID type in wallet API]
-    = "api" :> "new" :> "contract" :>
-        ("activate" :> ReqBody '[ JSON] (ContractActivationArgs t) :> Post '[JSON] ContractInstanceId -- start a new instance
+type API t walletId -- see note [WalletID type in wallet API]
+    = "api" :> ("healthcheck" :> Get '[JSON] ()) -- Is the server alive?
+    :<|> ("fullreport" :> Get '[JSON] (FullReport t)) -- Details of the contracts: the signatures and their states.
```

Comment:
if I read this correctly it produces

```
GET /api/healthcheck
GET /fullreport
POST /new/contract/activate
...
```

We can delete the "new" bit now and group everything under "api"

```
GET /api/healthcheck
GET /api/fullreport
POST /api/contract/activate
...
```

Reply:
Ah yes of course; nice, thanks I'll fix that.

Reply:
Thanks for showing me that `layout` trick for seeing the paths!

It's now:

```
/
└─ api/
   ├─ contract/
   │  ├─ activate/
   │  │  └─•
   │  ├─ definitions/
   │  │  └─•
   │  ├─ instance/
   │  │  └─ <capture>/
   │  │     ├─ endpoint/
   │  │     │  └─ <capture>/
   │  │     │     └─•
   │  │     ├─ status/
   │  │     │  └─•
   │  │     └─ stop/
   │  │        └─•
   │  └─ instances/
   │     ├─ wallet/
   │     │  └─ <capture>/
   │     │     └─•
   │     └─•
   ├─ fullreport/
   │  └─•
   └─ healthcheck/
      └─•
```